babel.config.js

```diff
@@ -2,7 +2,13 @@
 module.exports = {
   presets: [
     '@babel/preset-react',
-    '@babel/preset-env',
+    [
+      '@babel/preset-env',
+      {
+        useBuiltIns: 'usage',
+        corejs: '3.11',
```

Comment:
Do we have to specify version here? Can we just constrain it to the version we're running?

Reply:
Yeah I think it's needed: https://babeljs.io/docs/en/babel-preset-env#corejs